src/server/services/csvPipeline/validateCsvUploadParams.js

```diff
@@ -92,12 +92,16 @@ const DEFAULTS = {
 }
 
 // These are the common upload params shared between meters and readings upload.
-// Note: Even though, the 'email' and 'password' properties are not used to validate the upload, 
+// Note: Even though, the 'username' and 'password' properties are not used to validate the upload, 
 // they may be attached to the request body when the CSV Pipeline handles user authentication 
 // (i.e. when the user performs a curl request to the pipeline). Thus, we list these properties 
 // here so that they do not falsely trigger the 'additionalProperties' User Error.
 const COMMON_PROPERTIES = {
 	meterName: new StringParam('meterName', undefined, undefined),
+	username: new StringParam('username', undefined, undefined),
+	// allowing for backwards compatibility to allow for curl users to use the 'email' parameter instead of
```

Comment:
First, add TODO per other comment. Also, what happens if someone passes both the username and email? For example, which one is used?

Reply:
This takes the username param if it exists, otherwise it takes the email param. It only fails if neither is supplied. I tested with both in using curl and it works as intended.